resources/packs/azure/azure.lr

```diff
@@ -448,8 +448,33 @@ private azure.sql.server @defaults("id name location") {
   securityAlertPolicy() dict
   // SQL server encryption protector
   encryptionProtector() dict
+  // SQL server threat detection policy
+  threatDetectionPolicy() dict
+  // SQL server vulnerability assessment
+  vulnerabilityAssessment() azure.sql.server.vulnerabilityassessment
```

Comment:
this field name indicated that I receive an assessment. Those should be `vulnerabilityAssessmentSettings` to match the wording that Microsoft uses, see https://learn.microsoft.com/en-us/azure/defender-for-cloud/sql-azure-vulnerability-assessment-enable?tabs=express